src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/FilingsController.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.companieshouse.overseasentitiesapi.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestAttribute;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import uk.gov.companieshouse.api.model.filinggenerator.FilingApi;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.overseasentitiesapi.exception.SubmissionNotFoundException;
+import uk.gov.companieshouse.overseasentitiesapi.service.FilingsService;
+import uk.gov.companieshouse.overseasentitiesapi.utils.ApiLogger;
+
+import java.util.HashMap;
+
+import static uk.gov.companieshouse.overseasentitiesapi.utils.Constants.ERIC_REQUEST_ID_KEY;
+import static uk.gov.companieshouse.overseasentitiesapi.utils.Constants.OVERSEAS_ENTITY_ID_KEY;
+import static uk.gov.companieshouse.overseasentitiesapi.utils.Constants.TRANSACTION_ID_KEY;
+
+@RestController
+@RequestMapping("/transactions/{transaction_id}/overseas-entity/{overseas_entity_id}/filings")
+public class FilingsController {
+
+    @Autowired
+    private FilingsService filingService;
+
+    @GetMapping
+    public ResponseEntity<FilingApi[]> getFiling(
+            @RequestAttribute("transaction") Transaction transaction,
```

Comment:
There's a new request constant for "transaction" now in the Constants class. Please reference and use that here.